lib/node_modules/@stdlib/stats/base/dists/frechet/kurtosis/benchmark/benchmark.js

```diff
@@ -32,17 +33,25 @@ var kurtosis = require( './../lib' );
 
 bench( pkg, function benchmark( b ) {
 	var alpha;
+	var len;
 	var m;
 	var s;
 	var y;
 	var i;
 
+	len = 100;
+	alpha = new Float64Array( len );
+	s = new Float64Array( len );
+	m = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		alpha[ i ] = ( randu()*20.0 ) + EPS;
```

Comment:
Could you please update and generate the random variates directly from `@stdlib/random/base/uniform`?